/**:
  ros__parameters:
    vehicle_shape_margin_m: 1.0
    # search configs
    theta_size: 144
    # costmap configs
    obstacle_threshold: 100
    # planner configs
    rrt_margin: 0.1
